File: INICIANTE/1047/main.js
function elapsed(startH, startM, endH, endM) {
    let startMinutes = 60 * startH + startM;
    let endMinutes = 60 * endH + endM;
    let elapsedM = endMinutes - startMinutes;
    if (elapsedM < 1) elapsedM += 1440;
    let elapsedH = parseInt(elapsedM / 60);
    elapsedM = elapsedM % 60;
    return [elapsedH, elapsedM];
}

let line = require('fs').readFileSync('/dev/stdin', 'utf-8').trim();
let [startH, startM, endH, endM] = line.split(' ').map(Number);
let [elapsedH, elapsedM] = elapsed(startH, startM, endH, endM);
console.log(`O JOGO DUROU ${elapsedH} HORA(S) E ${elapsedM} MINUTO(S)`);

File: INICIANTE/1047/main.py
def elapsed(start_h, start_m, end_h, end_m):
    start_minutes = 60 * start_h + start_m
    end_minutes = 60 * end_h + end_m
    if (elapsed_m := end_minutes - start_minutes) < 1:
        elapsed_m += 1440
    elapsed_h = elapsed_m // 60
    elapsed_m = elapsed_m % 60
    return elapsed_h, elapsed_m


start_h, start_m, end_h, end_m = map(int, input().split())
elapsed_h, elapsed_m = elapsed(start_h, start_m, end_h, end_m)
print(f'O JOGO DUROU {elapsed_h} HORA(S) E {elapsed_m} MINUTO(S)')
